src/mds/MDSAuthCaps.cc

```diff
@@ -63,7 +63,7 @@ struct MDSCapParser : qi::grammar<Iterator, MDSAuthCaps()>
       lexeme[lit("'") >> *(char_ - '\'') >> '\''];
     unquoted_path %= +char_("a-zA-Z0-9_./-");
     network_str %= +char_("/.:a-fA-F0-9][");
-    fs_name_str %= +char_("a-zA-Z0-9-_.");
+    fs_name_str %= +char_("a-zA-Z0-9_.-");
```

Comment:
I'm not sure why changing the order works. But it does fix the issue.

Reply:
... I actually tried that and it didn't help! Maybe I screwed up restarting the mgr. How annoying.